backend/migrations/20240406024211_create_organisations.sql

```diff
@@ -2,19 +2,22 @@ CREATE TABLE organisations (
     id BIGINT PRIMARY KEY,
     name TEXT NOT NULL UNIQUE,
     logo TEXT,
-    created_at TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP,
-    updated_at TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP
+    created_at TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP NOT NULL,
+    updated_at TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP NOT NULL
 );
 
-CREATE TABLE organisation_admins (
+CREATE TYPE organisation_role AS ENUM ('User', 'Admin');
```

Comment:
I'm assuming this will be expanded later for other society roles but currently it seems there's no way for a user to become a 'User' of an org and I reckon this 'User' option is simply redundant since anyone on Chaos has access to any campaign by any organisation as per the current design.